README.md

```diff
@@ -190,7 +190,24 @@ The following addresses are preconfigured for testing:  (first 10 addresses from
 | 0x529043886F21D9bc1AE0feDb751e34265a246e47 | ["consumer"]                    |
 | 0xe08A1dAe983BC701D05E492DB80e0144f8f4b909 | ["publisher"]                   |
 | 0xbcE5A3468386C64507D30136685A99cFD5603135 | ["consumer","publisher"]        |
-
+| 0xBE5449a6A97aD46c8558A3356267Ee5D2731ab5e | ["user","consumer"]             |
+
+The following addresses are pre-configured with user profiles:
+
+| Address                                    |  Domain                  |  Email                             | User Roles                         |
+| -----------------------------------------  | -------------------------|------------------------------------|------------------------------------|
+| 0xe2DD09d719Da89e5a3D0F2549c7E24566e947260 | "ocean.com"              | "emptyuser@ocean.com"              | ["consumer", "validated", "user"]  |
+| 0xBE5449a6A97aD46c8558A3356267Ee5D2731ab5e | "oceanprotocol.com"      | "emptyuser@oceanprotocol.com"      | []                                 |
```

Comment:
maybe this one can remove since already added another one in below with consumer role

Reply:
Thanks for the feedback, I've removed the duplicates